server/bastionlab_common/src/telemetry.rs

```diff
@@ -66,6 +69,7 @@ impl TelemetryEventProps {
             TelemetryEventProps::SendDataFrame { .. } => "send_data_frame",
             TelemetryEventProps::ListDataFrame { .. } => "list_data_frame",
             TelemetryEventProps::GetDataFrameHeader { .. } => "get_data_frame_header",
+            TelemetryEventProps::SaveDataframe { .. } => "save_data_frame",
```

Comment:
Did you check that the telemetry server understands this?

Reply:
I've added this in the telemetry server as well.